### DISCUSSION TITLE
Set Default Blog in Post Editor?

### DISCUSSION THREAD

**Discussion by katyggls** [AprilSylph/XKit-Rewritten#1320] [Suggestion Box]
I'm not sure if this is something that's possible for you all to do, but I'm wondering if it's possible for there to be an option, maybe in Tweaks or whatever to set a default blog in the post editor. Currently, the post editor defaults to whatever blog you last posted to, but this is annoying and has caused me to post to the wrong blog a lot. I always forget to check and change the blog to my main blog. I imagine other people have this problem as well.

**Comment by marcustyphoon** [comment]
Interesting idea! One thing that makes this a bit interesting from the XKit Rewritten standpoint is that none of our settings are really account-specific (you can log out of Tumblr and log into another account and your XKit Rewritten settings will still be valid, or you can even use Firefox container tabs to have multiple Tumblr accounts logged in at once). So the user interface in XKit Rewritten's settings would have to have, like, a text box where you could enter a blog name (or multiple?) and it would try to change the selected blog to one of those. Definitely an option, though not a super elegant one.

(I'm not sure we can actually interact with these at all, of course, so there is also that.)

**Comment by arcanetrivia** [comment]
For what it's worth, that behavior annoyed me too when I first switched to the new post editor and it took me a while to get used to it, with the predictable result of posting to the wrong blog. Now I think I've retrained myself to always check and not depend on "it defaults to main", but being able to force that might be handy.
